subcommands/pkg/build.go

```diff
@@ -84,6 +84,7 @@ func (cmd *PkgBuild) Execute(ctx *appcontext.AppContext, repo *repository.Reposi
 	}
 
 	make := exec.Command("make", "-C", datadir)
+	make.Env = os.Environ()
```

Comment:
is this really needed?  it should inherit the environment from the current one.  (or at least that's what the doc says)